ci_group/revolve2/ci_group/morphological_novelty_metric/_morphological_novelty_metric.py

```diff
@@ -9,116 +8,122 @@
 from ._coordinate_operations import coords_from_bodies
 from .calculate_novelty import calculate_novelty
 
+Orientations = list[list[tuple[float, float]]]
```

Comment:
you are introducing global state. can you these inside the function? there is no reason to have state persist outside of the function call

Reply:
i thought this makes it more readable. but since it s used by 2 seperate functions i just removed it and did a fully typed typehint.